config.ini

```diff
@@ -18,7 +18,9 @@ AcPosition=1
 NumberOfInverters=1
 
 # Which DTU to be used ahoy, opendtu or template
-DTU=opendtu
```

Comment:
This was lost by accident. I will re-add it.

Reply:
please use opendtu as default, then I will approve